pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -186,11 +196,39 @@ func (c *connectivityCheckController) Sync(ctx context.Context, syncContext fact
 		}
 	}
 
-	newChecks, err := c.podNetworkConnectivityCheckFn(ctx, syncContext)
+	var newCheckNames sets.String
+	if c.podNetworkConnectivityCheckFn != nil {
+		newCheckNames, err = c.handlePodNetworkConnectivityCheckFn(ctx, syncContext)
```

Comment:
any plan to retire this function once the transition is made?  I prefer to have a single path if possible.